common/utils.py

```diff
@@ -0,0 +1,87 @@
+import json
+
+import requests
+
+IGNORED_LIST = ["row_id", "row_created", "row_updated"]
```

Comment:
dont use ignored list to remove the columns for select query , use only column you need.